buildenv/jenkins/JenkinsfileBase

```diff
@@ -1596,7 +1596,9 @@ def archiveChildJobTap(childJobs, originalStatus = 'SUCCESS') {
 		forceCleanWS()
 		try {
 			def buildPaths = ""
-			childJobs.each {
+			childJobs.stream()
+					.sorted { Comparator.comparing { it.value.getRawBuild().getNumber() } }
```

Comment:
We should sort on the build name (getProjectName), not the build number.

Reply:
And in the grinder the 'name' fields are behind `getProjectName()`?
```
14:32:42      GeneratedJob{name='Grinder_testList_0'}
14:32:42      GeneratedJob{name='Grinder_testList_1'}
```

Reply:
What happens with an 11th child job (`_10`)?

Reply:
I have an update that I will push shortly that aims to have a fixed length suffix (use the same number of digits for `${i}` for all jobs) so we would have `_00`, ... `_10`, etc. when there are more than 10 jobs.

Reply:
See b46406b436fe637d89ed1d40cec7012588ed26c3.